<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Label_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>87</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: white;
	background-color: rgb(58, 134, 255);
	border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_gender">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>243</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_gender">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: white;
	background-color: rgb(58, 134, 255);
	border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string>성별</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_birth">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>374</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_birth">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: white;
	background-color: rgb(58, 134, 255);
	border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string>생년월일</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_pw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>477</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: white;
	background-color: rgb(58, 134, 255);
	border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="Txt_pw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>361</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_resist">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>630</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>aakar</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:0 rgba(10, 242, 251, 255), stop:1 rgba(224, 6, 159, 255));
	border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>사용자 등록</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
